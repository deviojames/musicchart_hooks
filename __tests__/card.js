import React from 'react';
import renderer from 'react-test-renderer';
import Card from '../src/Components/Cards';

test('Screen matches the snapshot', () => {
    const tree = renderer.create(<Card />).toJSON();
    expect(tree).toMatchSnapshot();
  });