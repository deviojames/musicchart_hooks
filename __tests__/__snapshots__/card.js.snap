// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen matches the snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "white",
        "borderRadius": 5,
        "flexDirection": "row",
        "height": 50,
        "marginBottom": 5,
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 10,
        "paddingBottom": 0,
        "paddingLeft": 0,
        "paddingRight": 0,
        "paddingTop": 0,
        "shadowColor": "rgba(0,0,0,0.3)",
        "shadowOffset": Object {
          "height": 0,
          "width": 0,
        },
        "shadowOpacity": 1,
        "shadowRadius": 2,
      },
    ]
  }
/>
`;
